pkg/execute/executor.go

```diff
@@ -149,6 +147,8 @@ type CommandContext struct {
 	CommGroupName   string
 	BotName         string
 	RawCmd          string
+	CleanCmd        string
+	WithClusterName string
```

Comment:
Let's rename that to `ProvidedClusterName`
```suggestion
	ProvidedClusterName string
```

it'll be more verbose. WDYT?